kubeflow/argo/prototypes/argo.jsonnet

```diff
@@ -4,7 +4,7 @@
 // @shortDescription Argo workflow engine
 // @param name string Name to give to the component
 // @optionalParam namespace string null Namespace to use for the components. It is automatically inherited from the environment if not set.
-// @optionalParam imageTag string latest Docker image tag to use. It is automatically inherited from the environment if not set.
+// @optionalParam imageTag string v2.1.1 Docker image tag to use. It is automatically inherited from the environment if not set.
```

Comment:
image parameters should always be the full image path i.e. registry and tag. Any logic to combine a tag and image name should happen upstream.

Reply:
Done.

Reply:
Can we delete the imageTag param now?

Reply:
Done